### ISSUE TITLE
libtiff v6 update

### ISSUE THREAD

**Issue by Servinjesus1** [wrcad/xictools#16]
MacPorts now installs libtiff v6 by default, however xictools is still pointing to v5. This leads to an issue with dydl unable to point to the right library

(for Googlers, here is the specific issue):
```
dyld[6856]: Library not loaded: /usr/local/opt/libtiff/lib/libtiff.5.dylib
Referenced from: /usr/local/Cellar/sumo/1.17.0/bin/sumo
Reason: tried: '/usr/local/opt/libtiff/lib/libtiff.5.dylib' (no such file), ...
```

The issue occurs because `libtiff.6.dylib` is now installed, rather than 5.

## Current Workaround

`ln -s libtiff.6.dylib libtiff.5.dylib` is sufficient to fix this, as the libraries are similar enough to at least get Xic to load. Without knowing what this library does for Xic, however, I can't further test if it is working as expected.

## Issue Request

If it's possible to update XicTools to use libtiff v6, that would be great. Otherwise the Mac install instructions may require an additional note :)

**Comment by wrcad** [comment]
Yes, I got behind on the packages.  Probably it is best to build the programs from source as it seems impossible to keep up with OS changes and permutations.  The packages will probably go away eventually, to be replaced with a build script that will download what is needed and build from source.

**Comment by kev-cam** [comment]
Hi All,

Anyone interested in doing mixed signal simulation with Verilator?

https://cameron-eda.com/2020/06/03/rolling-your-own-ams-simulator/

and/or incorporating OpenVAF?

There's a market in power simulation displacing LTspice and SIMPLIS on
Windows.
Regards,
Kev.
